uber/site_sections/preregistration.py

```diff
@@ -453,6 +453,9 @@ def transfer_badge(self, session, message='', **params):
     def confirm(self, session, message='', return_to='confirm', **params):
         attendee = session.attendee(params, bools=_checkboxes, restricted=True)
 
```

Comment:
Ah, interesting point.  We could still accomplish that by including the ``id`` in the query string of the ``invalid_badge`` page (thus allowing us to see the id they're talking about), or even displaying ``cherrypy.request.headers[`Referer']`` in the template so they can copy/paste it to us.

As long as this is the only place where this is used, it doesn't matter much either way, of course.